content/en/docs/Reference/auroraboot.md

```diff
@@ -706,3 +706,76 @@ And then run:
 ```bash
 docker run -v "$PWD"/aurora.yaml:/aurora.yaml --rm -ti --net host quay.io/kairos/auroraboot /aurora.yaml
 ```
+
+
+## Booting with UKI and Secure Boot via EFI Key Enrollment
+
+AuroraBoot now supports booting **Unified Kernel Images (UKI)** with **Secure Boot**. This process relies on enrolling platform keys at boot time via a helper binary (`efi-key-enroller`) and involves a two-stage PXE boot process. This feature is useful for environments enforcing Secure Boot, allowing boot and installation of Kairos from a signed UKI ISO.
+
+> **Note**: Currently, only PXE boot has been tested and validated for this setup. HTTP boot support exists but has not been thoroughly tested.
+
+<p align="center">
+  <img src="/images/uki-secure-boot-flow.png" alt="PXE UKI Secure Boot Flow" style="max-width: 50%; height: auto;">
+  <br>
+  <em>PXE UKI Workflow</em>
+</p>
+
+## How it Works
+
+### Stage 1: PXE Boot and Key Enrollment
+
+1. The machine boots via PXE and executes `efi-key-enroller`.
+2. The enroller:
+    - Enrolls the provided `PK`, `KEK`, and `db` EFI keys (in `.auth` format).
+    - Creates an EFI boot entry pointing to the Kairos UKI ISO.
+    - Sets two EFI variables:
+        - One marking this as a PXE boot (`kairos.pxe=1`).
+        - One storing the ISO URL for use by the agent.
+
+3. The machine reboots and boots into the UKI ISO.
+
+### Stage 2: Kairos Agent Installation
```

Comment:
Maybe mention why it doesn't boot the efi-key-enroller again. (not in "setup mode" anymore, efi-key-enroller is not signed, etc).